rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NodeSource.java

```diff
@@ -433,6 +464,24 @@ public int hashCode() {
                 return nodeUrl != null ? nodeUrl.hashCode() : 0;
             }
 
+            public boolean checkNodeDetailsEquals(Object o) {
+                if (o == null || getClass() != o.getClass())
```

Comment:
Same thing, always use accolades for embedded code blocks, even if it is one line